lib/automate_liveness_agent/config.rb

```diff
@@ -78,6 +93,10 @@ def load_config_file
       apply_config_values(config_data)
     end
 
+    def setup_logger
```

Comment:
Having separate `setup_logger` and `logger` methods really only buys you savings if you don't want to initialize it all the time, but since it our logger we do. Having it as two methods requires the overhead of having to explicitly initialize it, otherwise we'll get nil errors. Might make sense to just make this method `logger` and remove the `attr_reader`.